focal-rt-ros2/rootfs/setup/phase1.sh

```diff
@@ -48,6 +48,10 @@ cp /boot/initrd.img /boot/firmware/initrd.img.bak
 systemctl disable ondemand
 systemctl enable cpu-frequency
 
+
+# Disable unattended-upgrades
+sudo dpkg-reconfigure -plow unattended-upgrades -fnoninteractive
```

Comment:
```suggestion
dpkg-reconfigure -plow unattended-upgrades -fnoninteractive
```
Let's remove it then